contrib/largeNbDicts/largeNbDicts.c

```diff
@@ -703,6 +704,22 @@ static int benchMem(slice_collection_t dstBlocks,
     }
     DISPLAY("\n");
 
+    char* csvFileName = malloc(strlen(exeName) + 5);
+    strcpy(csvFileName, exeName);
+    strcat(csvFileName, ".csv");
+    FILE* csvFile = fopen(csvFileName, "r");
+    if (!csvFile) {
+        csvFile = fopen(csvFileName, "wt");
+        assert(csvFile);
+        fprintf(csvFile, "%s\n", exeName);
+    } else {
+        csvFile = fopen(csvFileName, "at");
```

Comment:
Does this leak a `FILE` handle?

Reply:
I have it set up to write the executable name when the file doesn't already exist. That's why I have a branch. Agree that it's leaking the first `FILE` handle from `fopen(csvFileName, "r")` though, nice catch.